open_spiel/games/first_sealed_auction.h

```diff
@@ -83,7 +83,7 @@ class FPSBAGame : public Game {
     return std::unique_ptr<State>(new FPSBAState(shared_from_this()));
   }
   int MaxChanceOutcomes() const override {
-    return std::max(max_value_, num_players_);
+    return std::max(max_value_+1, num_players_);
```

Comment:
Based on the implementation in `ChanceOutcomes`:

https://github.com/deepmind/open_spiel/blob/8c8e0863380acaf93fe0b00780196d9117eaac66/open_spiel/games/first_sealed_auction.cc#L195

and 

https://github.com/deepmind/open_spiel/blob/8c8e0863380acaf93fe0b00780196d9117eaac66/open_spiel/games/first_sealed_auction.cc#L189

the range should be `[0, max(max_value_+1, num_players)]`.

Reply:
Will ask the author to take a look, thanks.

Reply:
@jkramar wdyt?

Reply:
As defined on line 189, the valuations are in the range `[1, max_value_]`, so I think `MaxChanceOutcomes()` as written is correct. How do you see `max_value_ + 1` arising?

Reply:
`max_value_ + 1` will not arise. But since `max_value_ ∈ [1, max_value_]` and `player ∈ [0, num_players_-1]`, so the possible chance outcome might be in range `[0, max(max_value_, num_players-1]`. Then I think `MaxChanceOutcomes()` is `max(max_value_+1, num_players_)`

Reply:
Icwym. But I think that's answering a different question: `MaxChanceOutcomes()`'s doc comment is that it's `// Maximum number of chance outcomes for each chance node`, whereas it seems you're using it analogously to `NumDistinctActions()`, which includes actions combined from different nodes/states (as you're doing). If this is how `MaxChanceOutcomes()` is used then I think the documentation should be updated to clarify that. Alternately you might use a separate function like `NumDistinctChanceOutcomes()`. @lanctot, wdyt ?

Reply:
We should be treating `MaxChanceOutcomes` as the chance node equivalent of `NumDistinctActions`, that was always my intention... apologies @jkramar if the documentation implies otherwise. @findmyway could you also fix the doc of the function to clarify this as well?

Reply:
Updated in [the latest commit](https://github.com/deepmind/open_spiel/pull/288/files/94d62eac2c8ba9223bb02060c7ee516e3e025f04..6794f2be80a5969643c4d1abcad6810ff170a4d7#diff-bdb8eda5e3f025cd1d06a389c4c99ee5L769).